test/test_prototype_transforms_consistency.py

```diff
@@ -840,3 +840,681 @@ def test_aa(self, inpt, interpolation):
         output = t(inpt)
 
         assert_equal(expected_output, output)
+
+
+class TestRefDetTransforms:
+    def make_datapoints(self, with_mask=True):
+        size = (600, 800)
+
+        pil_image = PIL.Image.new("RGB", size[::-1], 123)
+        target = {
+            "boxes": make_bounding_box(image_size=size, format="XYXY", extra_dims=(22,), dtype=torch.float),
+            "labels": features.Label(torch.randint(0, 81, size=(22,))),
```

Comment:
`22` seems to be the batch size here, correct? Can we please have this as a variable instead of hardcoding it multiple times? Maybe even call it `extra_dims` since that is what the parameter is called in the common utilities to generate the features.

Reply:
It is a number of objects, see that bbox has also 22